src/demos.py

```diff
@@ -83,15 +84,22 @@ def db_demo():
 
 def pipeline_demo():
     steps = [EnrichCustomDomains()]
-    input_location = "src/data/sumup_leads_email.csv"
-    output_location = "src/data/leads_enriched.csv"
+    input_location = "data/sumup_leads_email.csv"
+    output_location = "data/leads_enriched.csv"
```

Comment:
For me this caused an error as the program couldn't find the file. 
Inside of the Dockerfile if we want to remove the redundant src/ directory I think we need to adjust line 12 to be 
```
ADD src .
```
and line 15 to be
```
CMD [ "python", "main.py" ]
```

Furthermore, I would adjust the main to the following to ensure the app will always evaluate paths relative to the main.py location.

```[python]
import os

from demos import bdc_demo, db_demo, evp_demo, pipeline_demo

abspath = os.path.abspath(__file__)
dname = os.path.dirname(abspath)
os.chdir(dname)

if __name__ == "__main__":
    while True:
        try:
            choice = int(input("(1) BDC\n(2) EVP\n(3) DB\n(4) Pipeline\n(5) Exit\n"))
            match choice:
                case 1:
                    bdc_demo()
                case 2:
                    evp_demo()
                case 3:
                    db_demo()
                case 4:
                    pipeline_demo()
                case 5:
                    break
        except ValueError:
            print("Invalid choice")
```

Reply:
Addressed. Thanks for testing this!